<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWidget</class>
 <widget class="QWidget" name="FindWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="searchlayout">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/images/closesearch.png</normaloff>:/images/closesearch.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSearch"/>
     </item>
     <item>
      <widget class="QPushButton" name="searchPreviousButton">
       <property name="text">
        <string>Previous</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/images/findPrevious.png</normaloff>:/images/findPrevious.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchNextButton">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/images/findNext.png</normaloff>:/images/findNext.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="caseSensitiveCBox">
       <property name="text">
        <string>Case sensitive</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wholeWordsCBox">
       <property name="text">
        <string>Whole words</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editSearch</tabstop>
  <tabstop>searchPreviousButton</tabstop>
  <tabstop>searchNextButton</tabstop>
  <tabstop>caseSensitiveCBox</tabstop>
  <tabstop>wholeWordsCBox</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
